scripts/exglobal_atmos_tropcy_qc_reloc.sh

```diff
@@ -11,157 +11,154 @@
 ############################################################################
 
 # Make sure we are in the $DATA directory
-cd "${DATA}"
+cd "${DATA}" || exit
```

Comment:
Should probably be `exit 1` so the J-Job handles the failure correctly.

Reply:
Done.